### DISCUSSION TITLE
Separate sidebar.mts and nav.mts

### DISCUSSION THREAD

**Discussion by s-theo** [vuejs/vitepress#2969] [Q&A]
Since I have too many articles, this causes `config.mts` to be very bloated,
Is there a way to create `sidebar.mts` and `nav.mts` to write the contents and then import them from `config.mts`

**Comment by brc-dd** [comment]
Simply export your sidebar/navbar entries from them and import it in your config. What's the issue here?

**Comment by changbb** [comment]
![image](https://github.com/vuejs/vitepress/assets/23628509/33f82feb-c790-4e6a-bab1-66959e0d1c6d)
请问下如何解决的，我是这样拆分的，但是我导出的变量会报错!
![image](https://github.com/vuejs/vitepress/assets/23628509/49c89c2b-3c9f-4a4c-b0b3-bfe1c04f3682)

**Reply by brc-dd** [reply]
That's not how you export things. Try any of the following:

```ts
const frontendMenu = [...]

export default frontendMenu

// import frontendMenu from './frontend.js'
```

```ts
export const frontendMenu = [...]

// import { frontendMenu } from './frontend.js'
```

```ts
export default [...]

// import frontendMenu from './frontend.js'
```

**Reply by changbb** [reply]
Thank you for your help. Following your method, the problem has been resolved!
